kolibri/deployment/default/settings/dev.py

```diff
@@ -4,7 +4,7 @@
 
 from .base import *  # noqa isort:skip @UnusedWildImport
 
-INSTALLED_APPS += ["rest_framework_swagger"]  # noqa
+INSTALLED_APPS = list(INSTALLED_APPS) + ["rest_framework_swagger"]  # noqa F405
```

Comment:
`INSTALLED_APPS` was a tuple

Reply:
Ah, right, the plugin settings modification coerces it to a tuple.